Dockerfile-k8s

```diff
@@ -0,0 +1,72 @@
+FROM ruby:3.2.2-slim-bullseye AS rubyimg
```

Comment:
Looks like this image is no longer available. I think you'll need to update this line to pull the 3.2.3 image once the Ruby version for the app gets updated.

Reply:
good point - the ruby version will need to be updated in lock step with the version in Dockerfile and the ruby code - good catch. @JoeTice - do we have a ticket on any of the teams' list to address the upgrade to 3.2.3 of ruby for vets-api?

Reply:
3.2.2 is still available. https://hub.docker.com/_/ruby/tags?page=1&name=3.2.2-slim